Packs/UnifiVideoNVR/Integrations/UnifiVideo/UnifiVideo.py

```diff
@@ -141,8 +158,16 @@
         )]
     return_results(results)
 
-if demisto.command() == 'unifivideo-get-snapshot-at-frame':
-    vc = cv2.VideoCapture('recording.mp4')  # pylint: disable=E1101
+if demisto.command() == 'unifivideo-get-snapshot-at-frame':  # TOFIX
```

Comment:
Did you fix this command?
If you did, please remove `# TOFIX`

Reply:
yes. It's fixed. Thanks